samples/send-receive-files/ess-http.conf

```diff
@@ -362,6 +362,11 @@ TraceRootPath ./trace1
 # Environment variable: MONGO_ALLOW_INVALID_CERTIFICATES
 # MongoAllowInvalidCertificates
 
+# MongoSleepTimeBetweenRetry specifies the time in seconds between each retry of updating data in mongodb
```

Comment:
We need this in the ESS?

Reply:
@dabooz No, this value will not be used in ESS. It is only used for mongoStorage, which is only for CSS